src/sql/src/func.rs

```diff
@@ -1634,6 +1640,31 @@ macro_rules! builtins {
     }};
 }
 
+/// Constructs builtin function map with potentially dangerous functions.
+macro_rules! builtins_dangerous {
+    {
+        $(
```

Comment:
Instead of a new macro, what about adding a block without `, $dangerous:expr` that is a wrapper for this new one? Then you wouldn't need to enable these for certain schemas.

I also recommend not using a boolean in the macro definition itself, because booleans are bad as parameters. What about `"func_name" => Scalar, Dangerous {`? That has the downside that we programmers must know that this exists and specifically tag things as dangerous instead of the compiler enforcing it. Hmm.

A `builtin_dangerous` macro that defaults to dangerous unless fns are marked with `NotDangerous` would make default to dangerous for this schema, so it'd be opt in.